src/main/java/org/mockbukkit/mockbukkit/block/state/SignStateMock.java

```diff
@@ -39,7 +39,7 @@ public class SignStateMock extends TileStateMock implements Sign
 	public SignStateMock(@NotNull Material material)
 	{
 		super(material);
-		checkType(material, Tag.SIGNS);
+		checkType(material, Tag.ALL_SIGNS);
```

Comment:
Wouldn't this mean that the normal sign state mock can now be the hanging sign type?

Reply:
Yes that's correct. If we look into the implementation an "hanging sign" extends sign.
That was the reason i changed that so when we call super no exception is thown.